scripts/concurrent_task_utils.py

```diff
@@ -135,18 +135,18 @@ def execute_tasks(tasks, semaphore=SEMAPHORE):
     _check_all_tasks(currently_running_tasks)
 
 
-def create_task(func, verbose, semaphore=SEMAPHORE, name=None):
+def create_task(func, max_concurrent_runs, verbose=False, name=None):
     """Create a Task in its Thread.
 
     Args:
         func: Function. The function that is going to run.
         verbose: bool. True if verbose mode is enabled.
-        semaphore: threading.Semaphore. The object that controls how many tasks
-            can run at any time.
+        max_concurrent_runs: int. Number of tasks that run at a time.
```

Comment:
Args list is out of order with docstring. Please proofread.

Reply:
This was removed in c0ded58